e2e_tests/tests/cluster/test_checkpoints.py

```diff
@@ -64,7 +65,7 @@ def run_gc_checkpoints_test(checkpoint_storage: Dict[str, str]) -> None:
                 by_state.setdefault(checkpoint["state"], set()).add(checkpoint["total_batches"])
 
             if by_state == result:
-                all_checkpoints.append((config, checkpoints))
+                all_checkpoints.append((config, checkpoints, trials[0]["id"]))
```

Comment:
no need to put the same trial id in this data structure over and over.  There's only one trial in the experiment, just save it as a variable.